src/Dropbox/Models/AccessToken.php

```diff
@@ -60,6 +74,11 @@ public function __construct(array $data)
         $this->uid = $this->getDataProperty('uid');
         $this->accountId = $this->getDataProperty('account_id');
         $this->teamId = $this->getDataProperty('team_id');
+        $this->expiryTime = $this->getDataProperty('expires_in');
+
+        if ($this->getDataProperty('refresh_token')) {
```

Comment:
don't think this `if ($this->getDataProperty('refresh_token')) {` check is required

Reply:
:heavy_check_mark: